src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -0,0 +1,246 @@
+require "cwm/widget"
+
```

Comment:
I always make blank line to make a "border" between inmodule libs and outside ones, so cwm is outside, and y2partitioner is own.